marquez/airflow.py

```diff
@@ -1,93 +1,84 @@
 import json
 import pendulum
-import airflow.models
+from airflow.models import DAG, Log
 from airflow.utils.db import provide_session
 from marquez_client.marquez import MarquezClient
 
+from marquez.utils import JobIdMapping
 
-class MarquezDag(airflow.models.DAG):
+
+class MarquezDag(DAG):
+    _jobIdMapping = None
```

Comment:
consider using `_job_id_mapping` as it's more pythonic.

Reply:
sounds good!